blog/content/post/2020-06-15-qubes_summation.md

```diff
@@ -0,0 +1,184 @@
+Flicker session. Based on flicker, Piotr has proposed scenarios in which D-RTM
+may be used: Platform Relaunch, Virtual Machine Introspection technique and
+D-RTM, vTPM and D-RTM, Network booted vDLME and Visual Trust level indicator for
+Vms. The presentation has ended with future ideas, among others trusted system
+backups and migration, trusted firmware update, dynamic RPC policy or secure
+storage. The broad discussion covered among others the best way to re-establish
+trust in the platform.
+
+The second prelegent, Michał Żygowski has presented an Anti-Evil-Maid for Intel
+coreboot-based platform. Michał described what is needed for AEM to work on
+Intel processors, how Qubes OS Anti-Evil Maid works, what are instalation steps,
+what troubleshooting steps he himself had to go through to make AEM work, what
+is Intel TXT status in coreboot and how to enable Intel TXT on other hardware.
+
+### Qubes OS testing and development – by Qubes core developers
+
+On the last day of minisummit Qubes core developers took the floor. Frédéric
+Pierret, general packaging, CentOS and Fedora templates maintainer, explained
+how to build Qubes: from components to operating system overview. He desribed
+what is Qubes OS composed of, in terms of developing qubes (UX, Qubes, Isolation
+Provider Layer), presented the choice of Fedora as dom0 and VMs side
+distributions overview. In the next step, Frederic has explained how Qubes OS
+developers are introducing new features, fixes, what tools do they use and
+develop, on the example of Qubes builder v.1 and v.2.
+
+The last prelegent, Marek Marczykowski-Górecki, Project lead in QubesOS has
+introduced operating system testing, when it itself uses virtual machines. Marek
+briefly presented how to test Qubes OS, by describing python unit tests and
+pytest frameworks, used inside Qubes OS with Travis CI. He has also mentioned
+fuzzing tests, build tests, integration tests, installation tests, presented
+openQA tool used for running tests in VM and described missing parts of testing
+Qubes. During Q&A session we were considering how the community can help
+extending test scope and what kind of test are the most necessarry for Qubes OS
+development.
+
+
+## Summary
+
+ Qubes OS & 3mdeb 'minisummit' has been a great event that triggered many
+important questions and allowed the community to take an active part in the
+broad discussion around covered issues. We would like to thank Marek and
+Frédéric for sharing your knowledge and time through all the events. Thank you
+for your cooperation and willingness to ask and answer. Great appreciation goes
+towards all the community that joined our meetings and many times convinced us
+that events such as 'minisummit' are important and necessary. We have exchanged
+loads of knowledge and experience and we would like to meet again, on Qubes OS
+and 3mdeb 'minisummit' 2021. Thank you all once again.
+
+ If you think we can help in improving Qubes OS support for your hardware, help
```

Comment:
a/a

Reply:
I would rather leave this space here

Reply:
Why? I don't recall any blog post having the space here